detox/src/android/expect.test.js

```diff
@@ -209,27 +210,32 @@ describe('expect', () => {
   });
 
   describe('element screenshots', () => {
+    const tempFilePath = '/path/to/temp-file.ext';
     const invokeResultInBase64 = 'VGhlcmUgaXMgbm8gc3Bvb24h';
 
-    let ezFileIo;
+    let tempfile;
+    let fs;
     let _element;
     beforeEach(() => {
       mockExecutor.executeResult = Promise.resolve(invokeResultInBase64);
 
-      ezFileIo = require('../utils/easy-file-io');
-      ezFileIo.saveRawBase64Data.mockReturnValue('/mock/file/path');
+      tempfile = require('tempfile');
+      tempfile.mockReturnValue(tempFilePath);
+
+      fs = require('fs');
 
       _element = e.element(e.by.id('FancyElement'));
     });
 
     it('should take and save the screenshot in a temp-file', async () => {
       await _element.takeScreenshot();
-      expect(ezFileIo.saveRawBase64Data).toHaveBeenCalledWith(invokeResultInBase64, { fileSuffix: '.detox.elem-screenshot.png' })
+      expect(fs.writeFileSync).toHaveBeenCalledWith(tempFilePath, invokeResultInBase64, 'base64');
```

Comment:
Are you sure it is the best idea to block Node.js process? Use `fs-extra` and `await fse.writeFile()` maybe?

Reply:
Fixed!